build.bash

```diff
@@ -53,6 +55,7 @@ cat pages.json | jq --compact-output '.[]' | while read -r row; do
 	mkdir -p "$(dirname "${file}")"
 	"${install_dir}/lava" help "${topic}" > "${file}"
 	fix_headers "${file}"
+	fix_autolinks "${file}"
```

Comment:
Why don't you create a preprocessor instead?
https://rust-lang.github.io/mdBook/format/configuration/preprocessors.html

Reply:
I just followed the current approach. We can give a look to preprocessors in future itrations.